### PR TITLE
Add enrich signals mixin to block

### PR DESCRIPTION
**Pull request by hansmosh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +65,29 @@ one_value: False
   "shirt": [10, 14, 12],
   "group": ""
 }
+
```
Comment (kyliedale) [inline README.md]: This block of code doesn't end until after the next heading

File: README.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ If `one_value` is `True`, the output signal's attributes will each have a single
 
 If `group_by` is defined, an output signal will be produced for each value in the `group_by` attribute.
 
+If including existing signal attrinbutes with `enrich`, then the last signal passed into the block will be the one enriched.
```
Comment (kyliedale) [inline README.md]: Spelling on `attributes`

**Comment by kyliedale** [comment]
Couple of changes on the README, but the code looks good! Will merge after style changes 👍

**Comment by hansmosh** [comment]
@kyliedale: fixed, thanks!

**Pull request merged**